File: C#/백준/Bronze/16693. Pizza Deal/Pizza Deal.cs
using System;

namespace q{
    class w{
        static void Main(){
            double[] a = Array.ConvertAll(Console.ReadLine().Split(),double.Parse);
            double[] b = Array.ConvertAll(Console.ReadLine().Split(),double.Parse);
            double c1 = a[0]/a[1];
            double c2 = (b[0]*b[0]*Math.PI)/b[1];
            if(c1>c2){
                Console.WriteLine("Slice of pizza");
            }
            else{
                Console.WriteLine("Whole pizza");
            }
        }
    }
}

File: Python/백준/Bronze/16693. Pizza Deal/Pizza Deal.py
import math

a,b = map(int,input().split())
c,d = map(int,input().split())
c1 = a/b
c2 = (c*c*math.pi)/d
if(c1>c2):
    print('Slice of pizza')
else:
    print('Whole pizza')